Umweltzone/src/main/res/values-pt/strings.xml

```diff
@@ -0,0 +1,327 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  Copyright (C) 2020  Tobias Preuss
```

Comment:
Please change to `Copyright (C) 2020  Romero Silva`.

Reply:
Done!